netsim/devices/linux.yml

```diff
@@ -9,6 +10,8 @@ features:
       ipv6: true
     server: true
     relay: true
+  lag:
```

Comment:
Current FRR: https://github.com/ipspace/netlab/blob/7d80e7cabd32bcd16ddc668d6ff662cdab63ffe2/netsim/devices/frr.yml#L23

I vaguely remember you suggesting to use the generic mechanism, looks like that change may have gotten lost somehow.

Current template manually loads the kernel module with custom parameters in case of a VM, could add ```kmods``` for lag (to both Linux and FRR)

Reply:
> Current FRR

... is exactly what you need. The mapping between **lag** module and the **bonding** driver is here:

https://github.com/ipspace/netlab/blob/dev/netsim/providers/clab.yml#L24

Reply:
Didn't you add ```kmods``` 4 months ago? https://github.com/ipspace/netlab/commit/4cf8c7b1d1dc13a32953111f610f7a85c785d42e

Reply:
I remember suggesting to remove it and put it back with lag support, but you refused arguing that we might forget to put it back…